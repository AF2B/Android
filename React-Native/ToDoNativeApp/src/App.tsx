import React, { Component } from 'react'
import { SafeAreaView, Text, StyleSheet } from 'react-native'

import styles from './styles/styles'

class App extends Component {
  render() {
    return (
      <SafeAreaView style={styles.container}>
        <Text style={styles.text}>Hello World</Text>
      </SafeAreaView>
    )
  }
}

export default App;