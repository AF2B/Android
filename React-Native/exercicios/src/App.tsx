import React, { View, StyleSheet } from 'react-native'
import { Component } from 'react'
import HelloWorld from './components/HelloWorld'
import MinMax from './components/MinMax'
import Random from './components/Random'
import RandomTwo from './components/RandomTwo'
import CountingTwo from './components/CountingTwo'
import ComponentOne, { ComponentTwo, ComponentThree } from './components/Multiples'

class App extends Component {
  render() {
    return (
      <View style={style.Container}>
        <HelloWorld />
        <ComponentOne />
        <ComponentTwo />
        <ComponentThree />
        <br />
        <MinMax min="10" max="15" />
        <MinMax min={5} max={8} />
        <br />
        <Random numberOne={0} numberTwo={10} />
        <RandomTwo numberOne={20} numberTwo={50} />
        <br />
        <CountingTwo />
      </View>
    )
  }
}

const style = StyleSheet.create({
  Container: {
    flexGrow: 1,
    justifyContent: 'center',
    alignItems: 'center',
    padding: 10,
    backgroundColor: '#F5FCFF'
  },
  Button: {

  },
  ButtonText: {

  }
})

export default App