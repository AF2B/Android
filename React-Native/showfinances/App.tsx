import React from "react";
import { ThemeProvider } from "styled-components/native";
import { 
  useFonts,
  Poppins_400Regular, 
  Poppins_500Medium, 
  Poppins_700Bold } from '@expo-google-fonts/poppins';
import AppLoading from 'expo-app-loading';

import { Dashboard } from "./src/screens/Dashboard";
import theme from "./src/global/styles/theme";

function App() {
  // if (!useFonts({ Poppins_400Regular, Poppins_500Medium, Poppins_700Bold })) {
  //   return <AppLoading />
  // }

  return (
    <ThemeProvider theme={theme}>
      <Dashboard />
    </ThemeProvider>
  )
}


export default App;