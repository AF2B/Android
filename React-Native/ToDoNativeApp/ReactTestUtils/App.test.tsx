import React from "react";
import App from "../src/App";
import renderer from "react-test-renderer";

describe("App", () => {
  it('should be rendered hello world correctly', () => {
    const app = renderer.create(<App />);
    expect(app).toMatchSnapshot('hello world');
  });
});