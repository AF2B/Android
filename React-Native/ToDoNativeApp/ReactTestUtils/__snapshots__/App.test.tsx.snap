// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should be rendered hello world correctly: hello world 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
        "marginHorizontal": 15,
        "marginVertical": 15,
      }
    }
  >
    Hello World
  </Text>
</RCTSafeAreaView>
`;
